src/modules/uuv_att_control/uuv_att_control_params.c

```diff
@@ -239,3 +239,12 @@ PARAM_DEFINE_FLOAT(UUV_THRUST_SAT, 0.1f);
  * @group UUV Attitude Control
  */
 PARAM_DEFINE_FLOAT(UUV_SP_MAX_AGE, 2.0f);
+
+/**
+ * Stick mode selector (0=Pitch/roll control, 1=Heave/sway control; roll/pitch leveled)
+ *
+ * @group UUV Attitude Control
+ * @min 0
+ * @max 1
+ */
+PARAM_DEFINE_INT32(UUV_STICK_MODE, 1);
```

Comment:
Default should be PARAM_DEFINE_INT32(UUV_STICK_MODE, 0);

Reply:
Sounds good to me as well :)